### PR TITLE
gui: Stop translating PACKAGE_NAME

### PR DESCRIPTION
**Pull request by maflcko**
Generally the package name is not translated, but the package description is. 

E.g. `GIMP` or `Firefox` are always called that way regardless of the system language. However, "`Firefox` webbrowser" or "`GIMP` image manipulation program" are translated.

### CODE REVIEW

**Comment by hebasto** [comment]
Strong concept ACK (https://github.com/bitcoin/bitcoin/pull/15894#discussion_r297326036).

**Comment by MarcoFalke** [comment]
The GUI never picked up the translation anyway, so there shouldn't be any user-facing changes other than in InitErrors (See the difference between "Bitcoin Core" and "Bitcoin Kern"):

Screenshot from 2019-06-26 11-23-15

**Comment by promag** [comment]
Concept ACK.

**Comment by laanwj** [comment]
Concept ACK

**Comment by practicalswift** [comment]
Concept ACK

**Comment by fanquake** [comment]
@MarcoFalke 

>  So I'd say to backport this change to 0.18.1

What would be the benefit of backporting this? Isn't it just a simplification to the code that wont have any user facing effects?

**Comment by MarcoFalke** [comment]
In 0.18.1 only the following instances are translated: `git grep '_(PACKA'`, but not the others.

**Comment by laanwj** [comment]
> What would be the benefit of backporting this? Isn't it just a simplification to the code that wont have any user facing effects?

The only user-visible impact would be the InitErrors, I guess. It also reduces the workload of translators a bit by removing one(!) translation message. I don't think it's strictly necessary for backport but as there's really no added risk it doesn't hurt either.

**Comment by hebasto** [comment]
Is the `qt: Run «make translate» in ./src/` commit (fab85208f678ba1be53bdb73a73ce3c5c937d448) a subject to review?
It seems it should be a part of the release process, right?

**Comment by MarcoFalke** [comment]
Yes, it is subject to review, since I changed the buildfile and reviewers will need to check that the new buildfile works as expected anyway (with or without my commit).

**Comment by hebasto** [comment]
Is any reason to keep both `PACKAGE_NAME` and `COPYRIGHT_HOLDERS_SUBSTITUTION`?

**Comment by MarcoFalke** [comment]
I changed it so that `COPYRIGHT_HOLDERS_SUBSTITUTION` is also no longer translated, but I kept it for now. Maybe @luke-jr knows more why it is needed.

**Comment by luke-jr** [comment]
Generally, `COPYRIGHT_HOLDERS_SUBSTITUTION` should be "Bitcoin Core" even for software that isn't named "Bitcoin Core".

Regarding the PR itself: Should we get input from translators who have been using translated package names for a long time?

**Comment by laanwj** [comment]
> Regarding the PR itself: Should we get input from translators who have been using translated package names for a long time?

It can't hurt to ask people's input, but I'm not sure this makes much of a difference. E.g. are there any other occurences of "Bitcoin core" in the translated messages that shouldn't be translated?

**Comment by MarcoFalke** [comment]
> are there any other occurences of "Bitcoin core" in the translated messages that shouldn't be translated?

It doesn't look like it:

```
git grep 'Bitcoin Core' src/qt/locale/|grep '<source>'|grep -v '<source>Bitcoin Core</source>'|wc
      0       0       0

File: src/util/system.cpp [hunk 1]
```diff
@@ -1188,10 +1188,11 @@ int GetNumCores()
 
 std::string CopyrightHolders(const std::string& strPrefix)
 {
-    std::string strCopyrightHolders = strPrefix + strprintf(_(COPYRIGHT_HOLDERS), _(COPYRIGHT_HOLDERS_SUBSTITUTION));
+    const auto copyright_devs = strprintf(_(COPYRIGHT_HOLDERS), COPYRIGHT_HOLDERS_SUBSTITUTION);
```
Comment (hebasto) [inline src/util/system.cpp]: nit:
```
const std::string copyright_devs = strprintf(_(COPYRIGHT_HOLDERS), COPYRIGHT_HOLDERS_SUBSTITUTION);
```

**Review by hebasto — APPROVED** [review]
ACK fa64b947bb3075ff8737656706b941af691908ab, I have not tested the code, but I have reviewed it and it looks OK, I agree it can be merged.

**Comment by fanquake** [comment]
Is this still a candidate for backport? It's likely missed `v0.18.1` unless we have an rc2. But could still be pulled into the `v0.18` branch if wanted.

**Comment by fanquake** [comment]
Bumped to 0.18.2. I've included this for backport in #16617.

**Pull request merged**